packages/sit-onyx/src/components/OnyxListbox/types.ts

```diff
@@ -17,6 +17,17 @@ export type OnyxListboxProps<TValue extends SelectionOptionValue = SelectionOpti
    * Current value / selected options.
    */
   modelValue?: TValue;
+  /**
+   * Whether to show a loading indicator.
+   */
+  loading?: boolean;
+  /**
+   * How to emit the `loadMore` event:
+   * - lazy: User scrolls to the end of the options
+   * - button: User clicks the load more button. If there are no more options to load,
+   *           set this property to `undefined` so the button is hidden accordingly.
```

Comment:
instead of setting it manually, the dev experience might be nicer to set the total num of items in the config option and onyx handles comparing them

Reply:
see other comment. I mean to add
```
 loading?: boolean | {
   /** disables the loading feature once all items have been loaded */
   numberOfOptions?: number
  ....
  ```

Reply:
Edit: the other comment was a misunderstanding on my part.

it might still be an idea to add the `numberOfOptions` thing as part of `ListboxLoadingMode` :)